virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc

```diff
@@ -17,10 +17,22 @@ The Peripheral Component Interconnect (PCI) passthrough feature enables you to a
 
 Cluster administrators can expose and manage host devices that are permitted to be used in the cluster by using the `oc` command-line interface (CLI).
 
-include::modules/virt-about-pci-passthrough.adoc[leveloffset=+1]
+[id="virt-preparing-nodes-for-gpu-passthrough"]
+== Preparing nodes for GPU passthrough
+
```

Comment:
You may want to add a short abstract on line 22 before going right into the subheading.